stablehlo/reference/Ops.h

```diff
@@ -46,8 +49,17 @@ Tensor evalSubtractOp(const Tensor &lhs, const Tensor &rhs, Type resultType);
 Tensor evalTanhOp(const Tensor &operand, Type resultType);
 Tensor evalTransposeOp(const Tensor &operand, ArrayRef<int64_t> permutation,
                        Type resultType);
+SmallVector<Tensor> evalWhileOp(ArrayRef<Tensor> operand, Region &cond,
+                                Region &body, Scope &scope);
 Tensor evalXorOp(const Tensor &lhs, const Tensor &rhs, Type resultType);
 
+/// Evaluates an mlir::Region `region` using the runtime values `runtimeArgs`
+/// corresponding to the arguments of the containing block, assuming that the
+/// region has only one block. Interprets the operations within the block and
+/// returns the runtime values for the terminator's arguments.
```

Comment:
Proposed rewording:

```
/// Evaluates an mlir::Region `region` using the runtime values `args`
/// corresponding to the arguments of the entry block of the region.
/// Interprets the operations within the entry block and returns the runtime 
/// values for the terminator's arguments.
/// Assumes that the region has only one block.
```

Feedback:
  1) Rename "runtimeArgs" to "args".
  2) Use the more idiomatic "entry block" to refer to the region's first block.